avm/res/network/front-door/main.bicep

```diff
@@ -101,7 +101,7 @@ resource avmTelemetry 'Microsoft.Resources/deployments@2024-03-01' = if (enableT
   }
 }
 
-resource frontDoor 'Microsoft.Network/frontDoors@2020-05-01' = {
+resource frontDoor 'Microsoft.Network/frontDoors@2021-06-01' = {
```

Comment:
If this PR needs to include the API version update, I recommend indicating it in the PR title.

Reply:
Done